QemuPkg/Library/DfciDeviceIdSupportLib/DfciDeviceIdSupportLib.inf

```diff
@@ -0,0 +1,44 @@
+## @file
+# DfciDeviceIdSupportLib.inf
+#
+# NULL Library for DfciDeviceIdSupportLib.
+#
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005

```

Comment:
nit: For a new lib, should we use a newer `INF_VERSION`?

Reply:
Fixed